modules/web-console/frontend/app/components/stacktrace-viewer-dialog/components/stacktrace-item/template.pug

```diff
@@ -18,16 +18,16 @@
 )
     .stacktrace-viewer__cause_message
         .stacktrace-viewer__expander
-            div(ng-switch='$ctrl.cause.expanded')
+            div(ng-switch='$ctrl.expanded')
                 svg(ng-switch-when='true' ignite-icon='collapse')
                 svg(ng-switch-when='false' ignite-icon='expand')
-        span(ng-class='{bold: $ctrl.cause && $ctrl.cause.expanded}') {{ $ctrl.cause.message }}
+        span(ng-class='{bold: $ctrl.cause && $ctrl.expanded}') {{ $ctrl.cause.message }}
 .stacktrace-viewer__trace(
-    ng-show='$ctrl.cause.expanded && $ctrl.cause.stacktrace'
+    ng-show='$ctrl.expanded && $ctrl.cause.stacktrace'
     ng-repeat='trace in $ctrl.cause.stacktrace track by $index'
 )
     .stacktrace-viewer__trace_message
-        span {{ trace.message }}
-.stacktrace-viewer__trace(ng-if='$ctrl.cause.expanded && !$ctrl.cause.stacktrace')
-    .stacktrace-viewer__trace_message(ng-style='{"padding-left": "60px"}')
+        span {{ trace }}
+.stacktrace-viewer__trace(ng-if='$ctrl.expanded && !$ctrl.cause.stacktrace')
+    .stacktrace-viewer__trace_message
         span Full stacktrace is available only in #[a(href="https://www.gridgain.com/resources/download" target="_blank") GridGain since 8.8.1] version
```

Comment:
Please move to translations if not done already.

Reply:
All texts have been moved to translations.